spec/models/card_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe Card do
+  it "Check translation of the card" do 
+    card = Card.create(original: "cherry", translated: "Вишня", review_date: "2015-04-10 00:00:00 UTC")
+    expect(card.check_translation("CheRrY")).to be true
+  end
+  it "Checking the simple example" do
+    card = Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-10 00:00:00 UTC" )
+    expect(card.check_translation("Hoogie code")).to be true
+  end 
+  it "Returns false for incorrect translation" do
+    card = Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-10 00:00:00 UTC")
+    expect(card.check_translation("Shit")).to be false
+  end
+
+  context "The translation is true" do
+    let(:card) {Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-28 00:00:00 UTC")}
```

Comment:
Line is too long. [117/80]<br>Space missing inside {.<br>Space missing inside }.